master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -113,3 +114,55 @@ func DeleteConfigPolicies(ctx context.Context,
 	}
 	return nil
 }
+
```

Comment:
`GetEnforcedConfig` is fine as a name. I think something like `GetConfigPolicyField` is more descriptive. When I first read the function name, I thought it was only for invariant configs. It also wasn't clear that it was fetching a single field, rather than a whole or partial config.

Reply:
Great point! Changed this